src/hooks/useSubscriptions.ts

```diff
@@ -176,7 +189,7 @@ export function useSupporterSubscription(account) {
     )
 
     return () => {
-      supporterSubscription.current.unsubscribe()
+      supporterSubscription.current = null
```

Comment:
I wonder what's the reason to do it this way instead of using the `unsubscribe` method?

Reply:
To use unsubscribe we need to pass the correct type to that ref. Setting that to null does the trick. Although for coerence I will rollback and add `any` to that ref.